test/fileSummaryV1/fileSummaryV1.test.js

```diff
@@ -14,6 +14,7 @@ const saKeys = require('../../fileSummaryV1/service_account_keys');
 const metadataApi = require('../../fileSummaryV1/metadata_api');
 const urlSigner = require('../../fileSummaryV1/urlSigner');
 const apiAdapter = require('../../common/api_adapter');
+const helpers = require('../../common/helpers');
```

Comment:
`convertToFileInfoResponse` function can be imported here instead of general `helpers` just like in other .js files

Reply:
done!